layouts/index.redirects

```diff
@@ -2,11 +2,19 @@
 {{/* cSpell:ignore wordmark */ -}}
 
 {{ range $p := .Site.Pages -}}
-{{ range .Aliases -}}
-{{ . | printf "%-35s" }} {{ $p.RelPermalink }}
+{{ with .Aliases -}}
+
+{{ range $alias := . -}}
+  {{ $alias | printf "%-35s" }} {{ $p.RelPermalink }}
 {{ end -}}
+
+{{ with $p.Params.manualLink -}}
+  {{ $p.RelPermalink }} {{ . }} 301!
 {{ end -}}
 
+{{ end }}{{/* with .Aliases */ -}}
+{{ end }}{{/* range $p */ -}}
```

Comment:
This code is a bit specialized to our current case where we have a placeholder page with aliases. I'll generalize it in a followup PR.

Reply:
I'll track this via #1914.